### ISSUE TITLE
gradle configuration cache support in 0.6.0-Beta

### ISSUE THREAD

**Issue by Chrisjc28** [Kotlin/kotlinx-kover#203]
It seems when targeting version 0.6.0-Beta and enabling gradle caching,  there is a problem when running the kover html report task it finds a large amount of problems storing the configuration cache.

Example message from project- 
```
5169 problems were found storing the configuration cache, only the first 4096 were considered, 31 of which seem unique.
```

Just raising this as it might be something that needs to be looked into to for people who use gradle caching

**Comment by shanshin** [comment]
Relates #144

**Comment by shanshin** [comment]
Closed as a duplicate of #144
